coins.json

```diff
@@ -1213,8 +1213,8 @@
     "symbol": "ROSE",
     "decimals": 9,
     "blockchain": "OasisNetwork",
-    "derivationPath": "m/44'/474'/0'/0'/0'",
-    "curve": "ed25519HD",
+    "derivationPath": "m/44'/474'/0'",
+    "curve": "ed25519",
```

Comment:
`ed25519HD` is the current curve only being used for oasis? should we delete code associated with `ed25519HD` ?

Reply:
As Tadej said, you can do it!